Cargo.toml

```diff
@@ -43,6 +49,9 @@ directories-next = "2.0"
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 web-sys = { version = "0.3", features = ["Window", "Storage"] }
 
+[patch.crates-io]
+binance-rs-async = { git = "https://github.com/Fogapod/binance-rs-async", rev = "fb2a1daf1063d3b7edb551d4dc8ef8d4d7197538" }
```

Comment:
this patch shouldn't be needed?

Reply:
it removes a few dependencies from that lib